macros/adapters/default/page_views/snowplow_page_views.sql

```diff
@@ -24,9 +24,16 @@
 {% set use_perf_timing = (var('snowplow:context:performance_timing') != false) %}
 {% set use_useragents = (var('snowplow:context:useragent') != false) %}
 
+-- we are using 1-days window allowing us cover most of sessions
+-- but model become much faster comparing to selecting all events
 with all_events as (
 
     select * from {{ ref('snowplow_web_events') }}
+    {% if is_incremental() %}
+    where collector_tstamp > (
+        DATEADD('day', -1, (select coalesce(max(max_tstamp), '0001-01-01') from {{ this }}))
```

Comment:
A 1-day "look-back" is reasonable here, since the vast majority (~99%) of events will arrive within 24 hours of firing. I think we should make the interval length a configurable variable, so that analysts can choose 3, 7, or 30 days if they find that some quirk of their users or collector setup requires a greater "look-back" window.

That would look like:
```
where collector_tstamp > (
        DATEADD('day', -1 * {{var('page_view_lookback_days')}}, (select coalesce(max(max_tstamp), '0001-01-01') from {{ this }}))
```
Where `page_view_lookback_days` is defined as a variable in `dbt_project.yml` with a default value. My personal instinct for the default is to look back 3 days, but open to your thoughts!